examples/voice_receive/go.mod

```
@@ -1,7 +1,7 @@
 module github.com/bwmarrin/discordgo/examples/voice_receive
 
 require (
-	github.com/bwmarrin/discordgo v0.21.1
-	github.com/pion/rtp v1.6.0
-	github.com/pion/webrtc/v3 v3.0.0-20200721060053-ca3cc9d940bc
+    github.com/bwmarrin/discordgo v0.23.1
```

We have recently released v0.24.0, so I think it would be fair to update it here as well
```suggestion
    github.com/bwmarrin/discordgo v0.24.0
```